IntegrationDevelopment/I310/REST.postman_collection.json

```diff
@@ -0,0 +1,658 @@
+{
+	"info": {
+		"_postman_id": "501b10bf-3d50-4b84-a839-1de02df7b858",
+		"name": "MyBIIntegrationREST",
+		"description": "The set of examples for the Data Retrieval Basics training course.",
```

Comment:
I would remove this description. I have not found descriptions anywhere else in these collections.
Alternatively you can add descriptions to all courses, but please use the correct course names.